docs/scalardb-supported-databases.md

```diff
@@ -4,15 +4,15 @@ ScalarDB supports the following databases and their versions.
 
 ## Amazon DynamoDB
 
-| Version | DynamoDB |
-|:------------------|:---------------|
-| **ScalarDB 3.10** | ✅ |
-| **ScalarDB 3.9** | ✅ |
-| **ScalarDB 3.8** | ✅ |
-| **ScalarDB 3.7** | ✅ |
-| **ScalarDB 3.6** | ✅ |
-| **ScalarDB 3.5** | ✅ |
-| **ScalarDB 3.4** | ✅ |
+| Version           | DynamoDB |
+|:------------------|:---------|
+| **ScalarDB 3.10** | ✅        |
+| **ScalarDB 3.9**  | ✅        |
+| **ScalarDB 3.8**  | ✅        |
+| **ScalarDB 3.7**  | ✅        |
+| **ScalarDB 3.6**  | ✅        |
+| **ScalarDB 3.5**  | ✅        |
+| **ScalarDB 3.4**  | ✅        |
```

Comment:
Thank you for taking the trouble to do that.
I applied the IntelliJ markdown formatting, which improved the readability, but it does not handle the spacing well, likely because of the ✅ character. It was still an improvement, so I kept it like this.

Reply:
No problem, and thank you for adjusting the spacing!